tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -1808,8 +1809,10 @@ public function resetSpreedAppData() {
 		$currentUser = $this->currentUser;
 		$this->setCurrentUser('admin');
 		$this->sendRequest('DELETE', '/apps/spreedcheats/');
-		foreach ($this->changedConfigs as $config) {
-			$this->sendRequest('DELETE', '/apps/provisioning_api/api/v1/config/apps/spreed/' . $config);
+		foreach ($this->changedConfigs as $appId => $configs) {
+			foreach ($configs as $config) {
+				$this->sendRequest('DELETE', '/apps/provisioning_api/api/v1/config/apps/' . $appId . '/' . $config);
+			}
```

Comment:
@nickvergessen do you think that will be necessary preserve previous state of config if is a config of other app? Example: Make GET to store the current value before do a POST to define the config value. If the current value is null, will make a DELETE, if not, will make a POST

Reply:
From my POV this is a nice gimmic, but you pay with time for it. On each set you make 2 requests instead of one. Not very helpful.